test/spec/algoliasearch.helper/setQueryParameter.js

```diff
@@ -20,13 +20,7 @@ test('setChange should change the current state', function() {
 });
 
 test('setChange should not change the current state: no real modification', function() {
-  var helper = algoliasearchHelper(fakeClient, null, {
-    // @TODO: at the moment we have to provide a default value for the page, otherwise
-    // some methods reset the page to 0 (because of the reset behavior). This is what
-    // happens: page omit -> page defined with 0. We'll fix those issues with a next PR
-    // that implements a proper reset with the updated structure of the `SearchParameters`.
-    page: 0
```

Comment:
in the other instances of this comment you removed the page 0 from the test, but you kept it here, is that expected?

Reply:
Yes it's expected. The test checks that a change does not occur between two exact same calls. The call that is used inside the test is `setPage(0)`. Now that we don't have default values anymore we have to provide a value for the initial state.